rasa/core/constants.py

```diff
@@ -81,3 +82,13 @@
 # Names of the environment variables defining PostgreSQL pool size and max overflow
 POSTGRESQL_POOL_SIZE = "SQL_POOL_SIZE"
 POSTGRESQL_MAX_OVERFLOW = "SQL_MAX_OVERFLOW"
+
+USER = "user"
```

Comment:
This is not a good name. From what I can gather from the implementation it should be something like `USER_STATE`?

Reply:
Might make sense to to prefix all these constants with `STATE` to clarify how they are used.

Reply:
added a comment so that where they're created it says what are they for. When they are used the context is well defined, so in my opinion adding words will just be a distraction